driver/terraform_download.go

```diff
@@ -173,23 +177,23 @@ func installTerraform(ctx context.Context, conf *config.TerraformConfig) (*goVer
 		//
 		// At this point we cannot guarantee compatibility of the latest Terraform
 		// version, so we will move forward with a safe fallback version.
-		log.Printf("[WARN] (driver.terrform) could not determine latest version "+
-			"of terraform using checkpoint, fallback to version %s", fallbackTFVersion)
+		logger.Warn("could not determine latest version of terraform using Checkpoint, fallback to fallback version",
+			"fallback_version", fallbackTFVersion)
 		v = goVersion.Must(goVersion.NewVersion(fallbackTFVersion))
 	}
 
 	if err := isTFCompatible(conf, v); err != nil {
 		return nil, err
 	}
 
-	// Create path if doesn't already exist
+	// Create path if one doesn't already exist
 	os.MkdirAll(*conf.Path, os.ModePerm)
 
 	installedPath, err := tfinstall.ExactVersion(v.String(), *conf.Path).ExecPath(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	log.Printf("[DEBUG] (driver.terraform) successfully installed terraform %s: %s", v.String(), installedPath)
+	logger.Debug("successfully installed terraform", "version", v.String, "install_path", installedPath)
```

Comment:
This still has the `v.String` instead of `v.String()`, so it's not printing out the version:

```
[DEBUG] driver.terraform: successfully installed terraform: version=0x20e0a60 install_path=/Users/...
```

Reply:
ah good catch, thanks!